.gitignore

```diff
@@ -14,6 +14,9 @@
 #################################
 /nbproject
 .idea
+/.project
```

Comment:
These are Eclipse paths. As the comment says "# IDE and editor specific files #" this should be the correct place for them.